packages/outputarea/style/base.css

```diff
@@ -169,11 +169,36 @@ body.lm-mod-override-cursor .jp-OutputArea-output.jp-mod-isolated::before {
   margin: 0;
 }
 
-.jp-ThemedContainer .jp-TrimmedOutputs pre {
-  background: var(--jp-layout-color3);
-  font-size: calc(var(--jp-code-font-size) * 1.4);
-  text-align: center;
-  text-transform: uppercase;
+.jp-TrimmedOutputs-button {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  padding: var(--jp-flat-button-padding);
+  margin: 8px 0;
+  min-height: var(--jp-flat-button-height);
+  font-size: var(--jp-ui-font-size1);
+  font-family: var(--jp-ui-font-family);
+  background-color: var(--jp-layout-color1);
+  border: 1px solid var(--jp-border-color2);
+  color: var(--jp-ui-font-color1);
+  border-radius: var(--jp-border-radius);
```

Comment:
It looks like this may not be strictly needed as we get that from `.jp-ThemedContainer button`:

```suggestion
```

<img width="345" height="336" alt="Image" src="https://github.com/user-attachments/assets/43bdc3dc-5987-4944-9d79-2a898ebbc394" />